modules/ROOT/pages/installing-mobile-services.adoc

```diff
@@ -10,16 +10,149 @@ NOTE: {org-name} recommends running OpenShift using the method described here. H
+are installed and are the right version.
++
+Once verified it prompts you for your Docker Hub login credentials, which
+must be valid for the installation to continue.
++
+The next set of values are optional and can be left with the default values:
++
+[source,bash,subs="attributes"]
+----
+DockerHub Tag (Defaults to {release-number}):
+DockerHub Organisation (Defaults to aerogearcatalog):
+Cluster IP (Defaults to 192.168.37.1):
+Wildcard DNS Host (Defaults to nip.io):
+----
++
+The first two values here are used to configure the service-catalog in the
+resulting cluster to look at other docker registries or tags to fetch the
+APBs from, but unless you are already familiar with editing these values,
+the defaults are the safest option.
++
+The following 2 options alter what IP address and wildcard DNS host you wish
+to use.
+
+. *Cluster IP*
++
+In order to allow communication between your mobile device and the cluster, the
+cluster IP must be in the same c-block as your local network.
++
+i.e. If your local router which both your workstation and mobile are connected to
+a router using the `10.0.0.x` c-block, then you should set the cluster IP to an IP
+in the `10.0.0.x` c-block too, such as `10.0.0.135`.
+
+. *Finishing the installation*
++
+The following installation can take a while the first time it runs as it pulls
+quite a few docker images.
++
+Once completed successfully, this will end with output similar to:
++
+----
+TASK [output-oc-cluster-status : debug] ******************************************************************************************************************************************************
+ok: [localhost] => {
+    "msg": [
+        "Web console URL: https://192.168.37.1:8443/console/",
+        "",
+        "Config is at host directory /var/lib/origin/openshift.local.config",
+        "Volumes are at host directory /var/lib/origin/openshift.local.volumes",
+        "Persistent volumes are at host directory /var/lib/origin/openshift.local.pv",
+        "Data is at host directory /path/to/mobile-core/ui/openshift-data"
+    ]
+}
+
+PLAY RECAP ***********************************************************************************************************************************************************************************
+localhost                  : ok=44   changed=17   unreachable=0    failed=0
+----
++
+Which signifies that your installation has completed successfully.
+
+. *Verification*
++
+To verify that your installation has completed succesfully, you can open your
+favourite browser and (accepting the self-signed certificate) you can browse
+to:
++
+----
+https://192.168.37.1:8443/console/
+----
++
+The developer login credentials are:
++
+----
+username: developer
+password: password
+----
++
+The admin login credentials are:
++
+----
+username: admin
+password: admin
+----
++
+Once logged in look for the "mobile" category across the top of the catalog package
```

Comment:
this is continuation of the verifying sub list
.. Click Add to Project and choose Browse Catalog from the options.
.. Check that there is a Mobile tab displayed.
.. If you do not see a Mobile tab, see Troubleshooting

Reply:
I'll review again, leave it for now